torchreid/optim/optimizer.py

```diff
@@ -4,6 +4,10 @@
 
 import warnings
 
+from nncf.torch.quantization.layers import SymmetricQuantizer
```

Comment:
we are using nncf as optional requirements
Do we have a way how to wrap these imports?

Reply:
for example, you may look in engine.py. We wrap some imports

Reply:
Please check